src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,76 @@
+import clsx from 'clsx';
+import React from 'react';
+
+import { Oval } from 'react-loader-spinner';
+import styles from './Loader.module.css';
+
+// TODO: some props are shadowed from react-loader-spinner. Improve?
+export type Props = {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+
+  /**
+   * Layout size of the loader. This affects the overall size and associated
+   * stroke width.
+   */
+  size?: 'sm' | 'md' | 'lg';
+
+  /**
+   * Aria label of the oval. Default is oval-loading. Will be overridden if ariaLabel is passed in props
+   * (Shadowed from OvalProps)
+   *
+   * @see https://mhnpd.github.io/react-loader-spinner/docs/components/oval
```

Comment:
Unfortunately these jsdoc tags are not super compatible with Storybook, so these urls are not visible in the docs tab of the story page.
![image](https://user-images.githubusercontent.com/86632227/206526684-4b10acf0-8249-4cea-8bec-c98eb2a39e34.png)

Reply:
Sad. While they won't show in Storybook (yet?), they do show up in the IDE, so an engineer can see the doc.s and have an active link while doing their work. 

<img width="581" alt="Screen Shot 2022-12-08 at 11 35 50 " src="https://user-images.githubusercontent.com/3056447/206550809-40072092-ca65-4014-8dca-5e03e5699dfd.png">

Reply:
Makes sense to me, thanks for addressing